templates/agent.docker.yaml

```diff
@@ -21,6 +21,12 @@ identities:
     dst_port: 443
   pushdown:
   - dns_pattern
+- node:
+    name: docker_auth_elastic
+    dns_pattern: docker-auth.elastic.co
```

Comment:
seems very elastic specific, and not docker-agent per se. It could be part of another template agent.docker-elastic